main/src/main/res/layout-land/unifiedmap_activity.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Please have a look at the CI lint result, eg. at https://ci.cgeo.org/job/cgeo-CI_commit-build/1657/androidLintResult/new/.
Also the `xmlns:tools` here is unnecessary.

Reply:
all three fixed with PR #14817